src/components/Navbar/styled.ts

```diff
@@ -4,16 +4,15 @@ import { usingTypography, usingColors } from 'src/hooks/theme';
 
 export const Wrapper = styled.nav`
   display: flex;
+  align-items: center;
   padding: 0 ${usingTypography((t) => t.scaleSpacing(7))}px;
   padding-top: ${usingTypography((t) => t.scaleSpacing(3))}px;
   padding-bottom: ${usingTypography((t) => t.scaleSpacing(1))}px;
   box-shadow: 0 1px 4px ${usingColors((c) => c.ON.PRIMARY)};
 `;
 
 export const Icon = styled.img`
-  padding: ${usingTypography((t) => t.scaleSpacing(4))}px 0 0;
-  width: 100px;
-  height: 60px;
+  width: 200px;
```

Comment:
Personal opinion: Logoen her er ganske mye større enn den forrige. Dersom width settes til ~150px vil den omtrent være like stor som den forrige logoen 🙂
```suggestion
  width: 150px;
```